db/migrate/20230430083730_create_routes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRoutes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :routes do |t|
+      t.string :route_source, null: false
```

Comment:
IMHO we can skip prefix of attributes which is same as the model name. 
Ex. we rename `route_source` to `source` only.